servicecontrol/logging.md

```diff
@@ -10,18 +10,52 @@ redirects:
 
 ### Logging
 
-Instances of the ServiceControl service write logging information and failed message imports to the file system.  
-Before v1.9 the logging level by default was set to `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding the following to the configuration file:
+Instances of the ServiceControl service write logging information and failed message import stack traces to the file system.  
+
+Before v1.9 the default logging level was `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding by adding the following to the `appSettings` section of the  configuration file:
```

Comment:
"In Versions 1.9 and below logging level is"

"however this level can be quite verbose" this is redundant

"In Versions 1.9 and above is Warn"

i would also break this into two version sub headings since it is ambiguous what this applies to "this level is also configurable by adding by adding "